### PR TITLE
feat: experiment on post card action buttons spacing

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Remove the border in-between the upvote and downvote.
- Move the placement of the action buttons on list mode within the same row as the image.
- Keep a space in between the upvote and downvote to avoid layout shift.

Previews:

Screenshot 2024-12-19 at 12 23 57 AM
Screenshot 2024-12-19 at 12 16 24 AM
Screenshot 2024-12-19 at 12 16 12 AM
Screenshot 2024-12-19 at 12 16 07 AM
Screenshot 2024-12-19 at 12 16 04 AM
Screenshot 2024-12-19 at 12 03 41 AM
Screenshot 2024-12-19 at 12 03 37 AM
Screenshot 2024-12-19 at 12 03 35 AM

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-688.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx [hunk 1]
```diff
@@ -60,28 +63,7 @@ const ActionButtons = ({
     await onDownvoteClick?.(post);
   };
 
-  const lastActions = (
```
Comment (sshanzel) [inline packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx:63]: We don't need this to be in a variable anymore. Just copy-pasted way below.

File: packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx [hunk 2]
```diff
@@ -110,15 +92,20 @@ const ActionButtons = ({
...
+                  'ml-1.5 tabular-nums',
+                  !post.numUpvotes && feedActionSpacing && 'invisible',
+                )}
                 value={post.numUpvotes}
               />
             ) : null}
           </Button>
         </SimpleTooltip>
-        <div className="box-border border border-surface-float py-2.5" />
+        {!feedActionSpacing && (
```
Comment (rebelchris) [inline packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx:106]: In your screenshots only 1 version doesn't have the border is that correct?

File: packages/shared/src/lib/featureManagement.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ export class Feature<T extends JSONValue> {
 }
 
 const feature = {
+  feedActionSpacing: new Feature('feed_action_spacing', false),
```
Comment (rebelchris) [inline packages/shared/src/lib/featureManagement.ts:22]: Can you extract at bottom as seperate export please 🙏

**Review by rebelchris — APPROVED** [review]
Minor request for experiment export.
And one small border thing.

Design looks good, be sure to validate on deployment it doesn't affect current somehow.

File: packages/shared/src/lib/featureManagement.ts (hunk 3 above)
Comment (sshanzel) [reply packages/shared/src/lib/featureManagement.ts:22]: Ah, yes. I'll move it quickly.

File: packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx]: Screenshot 2024-12-19 at 8 55 05 PM
Screenshot 2024-12-19 at 8 55 03 PM
Screenshot 2024-12-19 at 8 54 57 PM
Screenshot 2024-12-19 at 8 54 53 PM
Screenshot 2024-12-19 at 8 54 51 PM
Screenshot 2024-12-19 at 8 54 44 PM
Screenshot 2024-12-19 at 8 54 42 PM

**Pull request merged**